src/components/shared/TractionInput.vue

```diff
@@ -12,7 +12,7 @@
 -->
 
 <template>
-  <div class="flex flex-col" :class="$attrs.class">
+  <div class="flex flex-col w-full" :class="$attrs.class">
```

Comment:
Worth checking if this affects the styling in any other places as we use this component a lot.

Reply:
I had a quick look and it all seems fine. Please tell if you see something different.
I couldn't make the input to fill up the space in ONTFlowCell without doing this ( <slot> for <TractionFieldError>) .

Reply:
Should be able to bind use the vue3 class attribute binding. So adding `:class="w-full"` to the input fields you want to have at fall width. Not a problem though if it doesn't affect other places.

Reply:
I will check that as it seems to be a better option

Reply:
Tried it, strangely that's not changing anything

Reply:
Ok no worries, I think its fine as is.